src/Totem.Tests/Features/Contracts/CreateTests.cs

```diff
@@ -243,77 +241,6 @@ public void ShouldNotCreateWhenContractStringIsNotValidSchema()
             command.ShouldNotValidate("Contract must be defined as a valid OpenAPI schema.");
         }
 
```

Comment:
SHOULD: Add integration test(s) for creating a contract without Id, Timestamp, Type (null), and Namespace (null)

Reply:
Updated with f36a51801c338d9897cf50b74466bea3817029e8